misc/python/materialize/buildkite_insights/util/buildkite_api.py

```diff
@@ -46,7 +46,7 @@ def get(
         if isinstance(result, dict) and result.get("message"):
             raise RuntimeError(f"Something went wrong! ({result['message']})")
 
-        params["created_to"] = result[-1]["created_at"]
+        params["page"] = str(int(params.get("page", "1")) + 1)
```

Comment:
This seems to work in all paginated results in Buildkite while the created_to  is only possible in some specific API endpoints.